pkg/beacon/analyzer.go

```diff
@@ -305,6 +305,10 @@ func getTsHistogramScore(min int64, max int64, tsList []int64) ([]int64, []int,
 	// use timestamps to get freqencies for buckets
 	freqList, freqCount, total, totalBars, longestRun := createHistogram(bucketDivs, tsList)
 
+	// calculate first potential score
+	// coefficient of variation will help score histograms that have jitter in the number of
+	// connections but where the overall graph would still look relatively flat and consistent
```

Comment:
Have we considered throwing out the zero-valued bars for calculating the coefficient of variation? In code, we could accomplish this by creating a copy of `freqList` while skipping over any zeros in the list.

In Case 1: Uniform Distribution with Jitter and Gaps, the new score will only consider the time period from 3-20 whereas the cv score will consider the whole time period. I believe the CV score will perform better if we throw those zero-valued bars out for that score as well.

Reply:
I did consider that for this update and I agree with you. However, I wanted to run more tests before I could commit to that change. We should look at it for a future update.